src/extras/GPGPU.js

```diff
@@ -114,6 +114,27 @@ export class GPGPU {
             this.fbo.swap();
         });
     }
+
+    dispose() {
+        this.geometry.dispose();
+        delete this.geometry;
+
+        this.passes.forEach((pass) => {
+            pass.program.dispose();
+
+            Object.values(pass.uniforms).forEach((uniform) => {
+                if (typeof uniform.dispose === 'function') uniform.dispose();
+            });
```

Comment:
Not sure about your targets but I'd prefer to use stuff like optional chaining here -- `uniform.dispose?.()`.

Reply:
good question, it looks like it's supported in safari since March 2020, so two years seems like a decent buffer!